src/support/timefuncs.c

```diff
@@ -28,31 +28,32 @@
 extern "C" {
 #endif
 
-#if defined(_OS_WINDOWS_)
-static double floattime(void)
+int jl_gettimeofday(struct jl_timeval *jtv)
```

Comment:
good practice to have them consistent - msvc can occasionally fail to compile when they aren't